webpack.config.js

```diff
@@ -48,6 +49,17 @@ const moduleRules = [
             },
         }],
     },
+    {
+        test: /theme\.js$/i,
+        use: [{
+            loader: 'babel-loader',
```

Comment:
Babel is a fairly large package, I'm not explicitly against adding it, but if we do add it we should also look into adding it to the other rules. It's also a topic worth a discussion whether we should add it or not.